piety-promoted-v1/modernized/17-chapter-16.adoc

```diff
@@ -717,8 +718,12 @@ in Friends`' burying place, near Dolphin`'s Barn,
 being accompanied to the grave by many friends and neighbors.
 Aged near nineteen, and a minister about six years.
 
+[.embedded-content-document.testimony]
+--
+
+[.letter-heading]
```

Comment:
This doesn't seem as much like a Letter-heading to me.  I would probably take it out of the embedded section, and mark it as an offset. 👍🏻